misc/mutt2khal

```diff
@@ -37,7 +40,7 @@
 }
 
 END {
-  print "khal new", loc, d_st, t_st, d_end, t_end, summ, desc | "sh"
+  printf("khal new --location %s %sT%s %sT%s %s :: %s", loc, d_st, t_st, d_end, t_end, summ, desc) | "sh"
```

Comment:
The `T`s here will break the default configuration.

Reply:
Yes, but according to https://github.com/pimutils/khal/issues/1078#issuecomment-896695860 it is the only way to currently pass dates to khal.

Reply:
Okay, got it, khal only accepts the one date and time format specified in the config; I somehow thought it always tries to interpret all formats thrown at it (independent of the configuration).

Reply:
Corrected it  to assume the default format again.